src/indexApi.ts

```diff
@@ -95,11 +95,16 @@ function main(): void {
       // proper api arcitechture should page forward instead of all in 1 chunk
       limit: 1000000
     }
-    const result = asDbReq(
-      await reportsTransactions.partitionedFind(appAndPluginId, query)
-    )
+
+    let result
```

Comment:
check for what error, error 500 for couchdb being down, error 400 for invalid query or partition doesnt exist.  Test to make sure.